service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrganisationService.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityUserLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.exceptions.UnknownLocalAuthorityCodeException;
+import uk.gov.hmcts.reform.rd.client.OrganisationApi;
+import uk.gov.hmcts.reform.rd.model.Status;
+import uk.gov.hmcts.reform.rd.model.User;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+public class OrganisationService {
+    private final LocalAuthorityUserLookupConfiguration localAuthorityUserLookupConfiguration;
+    private final OrganisationApi organisationApi;
+    private final AuthTokenGenerator authTokenGenerator;
+
+    @Autowired
+    public OrganisationService(LocalAuthorityUserLookupConfiguration localAuthorityUserLookupConfiguration,
+                               OrganisationApi organisationApi,
+                               AuthTokenGenerator authTokenGenerator) {
+        this.localAuthorityUserLookupConfiguration = localAuthorityUserLookupConfiguration;
+        this.organisationApi = organisationApi;
+        this.authTokenGenerator = authTokenGenerator;
+    }
+
+
+    public List<String> findUserIdsInSameOrganisation(String authorisation, String userId, String localAuthorityCode) {
+        List<String> userIds = new ArrayList<>();
+
+        try {
+            addUsersFromSameOrganisationBasedOnAppConfig(localAuthorityCode, userIds);
+        } catch (UnknownLocalAuthorityCodeException ex) {
+            try {
+                addUsersFromSameOrganisationBasedOnReferenceData(authorisation, userIds);
+            } catch (Exception e) {
+                log.warn("Can't find LocalAuthority for code: "
```

Comment:
Use logging system support for interpolation  "... {} ..." instead of doing manual string concatenation.

Reply:
Can I do both exception and placeholders? The SLF4j log API, documentation informs me the arguments I am passing are just objects for placeholders, which makes me nervous.

Reply:
From my experience you can. Exception should be last arg.

Reply:
Ok thanks, I have refactored the code so this is no longer relevant, but will remember for the future.